Sources/NiceComponents/Helper/Font+NiceTextStyle.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Font+NiceTextStyle.swift
+//  NiceComponents
+//
+//  Created by Alejandro Zielinsky on 2022-06-30.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import SwiftUI
+
+public extension Font {
+    /// Create a custom Font from a given TextTheme
+    /// - Parameter fontStyle: The styling to use when creating a Font.
+    /// - Returns: A Font using the size and weight specified in your NiceTextStyle
+    static func custom(_ style: NiceTextStyle) -> Font {
```

Comment:
I know this is not new to this version, but public extensions on system types, in libraries, with fairly generic names, make me a little nervous. I wonder if this should maybe either have a more unique name that includes "nice" in it somewhere (`fromNiceStyle`?) or if this should be a method on NiceTextStyle instead.

Reply:
I see where you're coming from and totally agree, though it does seem like there's already [a number of .custom calls on Font](https://developer.apple.com/documentation/swiftui/font/custom(_:fixedsize:), is it less risky because we're passing in a custom type?

Reply:
You're right that it is less risky if it's only taking a custom type, because then it can't ever have an unresolvable conflict with another extension with the same name in a different library (which is the most serious risk). There's still some risk of a resolvable ambiguity (particularly if other things were adding similarly named extensions), and there's the possibility for confusion if people reading code using it don't understand that there's an extension being pulled in from a library, so I'd say it's slightly better to not do this in general, but I think you're right that in this specific case it is probably okay because of the custom type thing, so maybe not worth changing right now.